gobblin-core/src/test/java/gobblin/converter/filter/AvroSchemaFieldRemoverTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.converter.filter;
+
+import java.io.IOException;
+
+import org.apache.avro.Schema;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import gobblin.util.AvroUtils;
+
+
+/**
+ * Tests for {@link AvroSchemaFieldRemover}
+ */
+@Test(groups = { "gobblin.converter.filter" })
+public class AvroSchemaFieldRemoverTest {
+
+  private FileSystem fs;
+
+  @BeforeClass
+  public void setUp() throws IOException {
+    this.fs = FileSystem.get(new Configuration());
+  }
+
+  @Test
+  public void testRemoveFields() throws IllegalArgumentException, IOException {
+    Schema convertedSchema1 =
+        convertSchema("gobblin-core/src/test/resources/converter/recursive_schema_1.avsc", "results.activityInfo");
```

Comment:
This relies on the test being run from a specific directory (the root). That may vary depending on if we use Eclipse, IDEA, or the command-line. Since the avsc files is already in src/test/resources, it will be automatically included as a resource in the test jar. We should use ClassLoader.getResourceAsStream("converter/recursive_schema_1.avsc") to read the file instead.